### PR TITLE
docs: databend-driver for python

### PR DESCRIPTION
**Pull request by soyeric128**
Introduced [databend-driver](https://pypi.org/project/databend-driver/).

### CODE REVIEW

**Comment by soyeric128** [comment]
I drafted the code with `AsyncDatabendClient` and it worked fine from my computer. It would be helpful if you help to confirm these doubts:

- i noticed the connection code containing the parameter `sslmode`, what else parameters available here?
- Please give an example connecting to Databend Cloud. Thanks.

**Comment by everpcpc** [comment]
The connection params and DSN is the same with `BendSQL`

**Comment by hantmac** [comment]
I'm wondering whether we hidden `databend-py` directly or providing both `databend-py` and `databend-driver(python binding)`.  I think providing both options to the users would be better.

**Review by hantmac — APPROVED** [review]
/LGTM

File: docs/doc/03-develop/01-python.md [hunk 1]
```diff
@@ -37,75 +37,53 @@ Before you start, make sure you have successfully installed a local Databend. Fo
...
 2. Copy and paste the following code to the file `main.py`:
 
 ```python title='main.py'
-from databend_py import Client
+from databend_driver import AsyncDatabendClient
+import asyncio
 
-# Setting secure=False means the client will connect to Databend using HTTP instead of HTTPS.
-client = Client('user1:abc123@127.0.0.1', port=8000, secure=False)
+async def main():
```
Comment (hantmac) [inline docs/doc/03-develop/01-python.md:70]: The databend_driver currently only supports asynchronous mode. This aspect could be articulated more clearly in the documentation.

**Pull request merged**